pkgs/misc/wiki-tui/default.nix

```diff
@@ -19,25 +20,29 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-euyg4wYWYerYT3hKdOCjokx8lJldGN7E3PHimDgQy3U=";
   };
 
-  nativeBuildInputs = [
-    pkg-config
-  ];
+  nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [
     ncurses
     openssl
-  ] ++ lib.optionals stdenv.isDarwin [
-    Security
-  ];
+  ] ++ lib.optionals stdenv.isDarwin [ Security ];
 
-  cargoHash = "sha256-rKTR7vKt8woWAn7XgNYFiWu4KSiZYhaH+PLEIOfbNIY=";
+  # Note: bump `time` dependency to be able to build with rust 1.80, should be removed on the next
```

Comment:
Is this already done upstream or a Pr open for that? If not would you be so kind.

Reply:
(I'm the maintainer of the package) Upstream has a lot of changes already in `main`, most of which aren't compatible with 0.8.2 (changing the UI framework)

Reply:
> Is this already done upstream

[Looks like upstream no longer even depends on time.](https://github.com/Builditluc/wiki-tui/blob/main/Cargo.lock)